docs/src/api/class-apirequest.md

```diff
@@ -0,0 +1,53 @@
+# class: ApiRequest
+* langs: js
+
+Exposes API that can be used for Web API testing.
+
+## async method: ApiRequest.newContext
```

Comment:
how about getting rid of ApiRequest and replacing it with `playwright.newApiSession/Context` ?